akka-docs/src/main/paradox/java/stream/stages-overview.md

```diff
@@ -1 +0,0 @@
-../../scala/stream/stages-overview.md
```

Comment:
this doesn't work, as symlinks are relative so now "akka-docs/src/main/paradox/java/stream/stream-composition.md" is a symlink to "akka-docs/src/main/paradox/java/stream/akka-docs/src/main/paradox/scala/stream/stream-composition.md" which obviously doesn't exist :)

you can test symlinks simply by looking at the contents, i.e. "cat akka-docs/src/main/paradox/java/stream/stream-composition.md"